examples/snippets/tags/application.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Google LLC
+#
+# Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+require "io/console"
+require_relative "../config/environment"
+require_relative "models/singer"
+require_relative "models/album"
+
+class Application
+  def self.run
+    # Query all albums and include a request tag for the statement.
+    puts ""
+    puts "Query all Albums and include a request tag"
+    albums = Album.annotate("request_tag: query-all-albums").all
```

Comment:
Just curious: what will happen if `request tag` is `query all albums`?

Reply:
It will use `query all albums` as the request tag. We actually (by chance) use tags with spaces in the tests: https://github.com/googleapis/ruby-spanner-activerecord/blob/998bc5c30ddbfd7bf785e1a5727b3a04c1da06d7/test/activerecord_spanner_mock_server/spanner_active_record_with_mock_server_test.rb#L837